tests/unit_tests/test_app_data.py

```diff
@@ -0,0 +1,91 @@
+import time
+import unittest
+from datetime import datetime
+from unittest.mock import patch, Mock
+
+import pymongo.collection as collection
+import pymongo.results as results
+
+from app.data import MongoDB
+
+
+class TestMongoDBQueries(unittest.TestCase):
+
```

Comment:
Do these database tests bring value to our codebase? It looks like we're just testing if Mock works as intended.

Reply:
A subtle distinction, but the tests are technically checking the functions' extraction of "success" data from the response object-- it's just that the current code uses the simple "acknowledged" property as a stand-in for a more robust success/failure state determination. The argument to keep or discard these tests depends on whether this state might change, and whether it's worthwhile having a test foundation already in place for it.

Reply:
OK, so if I or a new team member refactors the update method (incorrectly) this test will detect that?

Reply:
It catches the marginal case where if you drop the `.acknowledged` or otherwise return the wrong response object from the function it will always read as trivially true whereas we might want the (limited) feedback from the write acknowledgement. It would be a bit more interesting if we had a db object set up with the more in-depth status feedback mongodb can provide, but as it stands it's very limited unfortunately.